patches/minecraft/net/minecraft/block/Block.java.patch

```diff
@@ -274,12 +274,16 @@
 +    }
 +
 +    /**
-+     * State-sensitive version of isVisuallyOpaque.
++     * State-sensitive version of func_176214_u.
++     * <p>
++     * This method is used for:
++     * <br> Checking if an entity should be suffocating
++     * <br> Checking if the suffocation overlay should be rendered in 1st person
 +     *
 +     * @param state The current state
-+     * @return True if the block is visually opaque
++     * @return True if the block can suffocate entities that are inside it.
 +     */
-+    public boolean isVisuallyOpaque(IBlockState state)
```

Comment:
This is the newly-added version that takes an `IBlockState` parameter. The original still exists and has been marked deprecated, see patch line 33.